eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -10,6 +10,7 @@ parameters:
   TargetDocRepoOwner: ''
   PRBranchName: 'smoke-test-rdme'
   SourceBranchName: 'smoke-test'
+  PRLabels: 'automation'
```

Comment:
@scbedd should we go ahead and also start applying the 'auto-merge' label?

Reply:
Sync with @scbedd , the DocMs PR does not have any `auto-merge` label created yet. Will have `auto-merge` as a placeholder for now, but it might not function like it should be. Meanwhile, I will have a simple test to verify the script did not fail if the label does not exist.

Reply:
After chatting with @scbedd I think we want to just replace 'automation' with 'auto-merge'.

Reply:
Yes, forgot to update. Thanks!